queries/lua/highlights.scm

```diff
@@ -172,7 +172,7 @@
 (function_call name: (dot_index_expression field: (identifier) @function.call))
 (function_declaration name: (dot_index_expression field: (identifier) @function))
 
-(method_index_expression method: (identifier) @method)
+(method_index_expression method: (identifier) @method.call)
```

Comment:
those two languages are not related it would have been better to have two PRs for things like that.

From my point the Lua part looks ok. I'm not sure about LaTeX. Can ` command_name` only occur in command invokation or also in command definition (probably not `command_name` there. @clason

Reply:
It's also used in definitions, so this needs to be made more targeted (distinguish based on `field`).